packages/backend/src/peripherals/database/migrations/004_fact_to_pages.ts

```diff
@@ -0,0 +1,28 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('fact_to_pages', (table) => {
+    table.increments('id').primary()
+    table.integer('block_number').notNullable()
```

Comment:
At some point we'll need to set up database indices on columns like `block_number`, but that can be done in the future